test/run.sh

```diff
@@ -207,8 +207,7 @@ if [[ -z $TRAVIS_CI ]] && [[ -f "$dir/local_test_env.sh" ]]; then
 	source "$dir/local_test_env.sh"
 fi
 
-BOULDER_IP="$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.Gateway}}{{end}}' boulder)" \
-	&& export BOULDER_IP
+BOULDER_IP="10.77.77.1" && export BOULDER_IP
```

Comment:
They modified the docker-compose file to use [two Docker networks with specific subnet (10.77.77.0/24 and 10.88.88.0/24)](https://github.com/letsencrypt/boulder/blob/8ebdfc60b694e7d5870c1ed1a089b10e38b5a338/docker-compose.yml#L69) instead of running on the default bridge network, so the previous method to retrieve the Boulder container's gateway IP now return two IPs, they are passed to the bash function that poll for the ACME endpoint and it times out.

I think I can come up with something a bit more robust if required.